### PR TITLE
apps picker: allow configurability of default app

### LINKED ISSUE #190: Keep "last client id created" in memory and offer that as the default for commands that require client ID [CLI-91]
image

Instead of having to find the client id, we can offer the last client created as the default option (just store it in a global var in memory). Anywhere where we ask for client ID we should default to that. Delighter.

### PR DESCRIPTION
**Pull request by cyx**
Closes #190

## Demo

[asciicast](https://asciinema.org/a/qrN8xhwE335nxAusohyJQtjuv)

### CODE REVIEW

**Comment by woloski** [comment]
🔥chef kiss

File: internal/cli/apps.go [hunk 1]
```diff
@@ -117,10 +124,60 @@ func appsCmd(cli *cli) *cobra.Command {
...
+
+			if err := cli.setDefaultAppID(inputs.ID); err != nil {
+				return err
+			}
+
+			if inputs.ID == "" {
+				cli.renderer.Infof("Successfully removed the default application")
+			} else {
+				cli.renderer.Infof("Successfully set the default application to %s", ansi.Faint(inputs.ID))
+				cli.renderer.Infof("%s: You might wanna try `auth0 quickstarts download`", ansi.Faint("Hint:"))
```
Comment (Widcket) [inline internal/cli/apps.go:170]: Can you please add the client id to the hint?

Comment (Widcket) [reply internal/cli/apps.go:170]: Also the hint is showing two colons after `Hint`.

Comment (cyx) [reply internal/cli/apps.go]: Done! Good eye, thanks 💪

**Review by Widcket — APPROVED** [review]
Suggested a fix for a typo, otherwise LGTM.

**Pull request merged**